src/esbmc/options.cpp

```diff
@@ -88,7 +88,14 @@ const struct group_opt_templ all_cmd_options[] = {
     {"no-visibility",
      NULL,
      "force to verify every function, even it's an unreachable "
-     "internal/private function"}}},
+     "internal/private function"},
```

Comment:
@can you move these options under the solidity frontend?

Reply:
Just checked.. It seems to be already under solidity option

![image](https://github.com/user-attachments/assets/0dc2fc6c-2a30-4be5-befd-49f8904c8c48)